packages/agent-toolkit/src/openai/toolkit.ts

```diff
@@ -1,14 +1,22 @@
 import { ApiClient, ApiClientConfig } from '@mondaydotcomorg/api';
-import { zodToJsonSchema } from 'zod-to-json-schema';
 import type {
-  ChatCompletionTool,
   ChatCompletionMessageToolCall,
+  ChatCompletionTool,
   ChatCompletionToolMessageParam,
 } from 'openai/resources';
 import { z } from 'zod';
-import { Tool, allTools } from '../core';
-import { filterTools, ToolsConfiguration } from '../core/utils';
+import { zodToJsonSchema } from 'zod-to-json-schema';
+import { Tool } from '../core/tool';
+import { allGraphqlApiTools, allMondayAppsTools } from '../core/tools';
+import { filterApiTools, filterMondayAppsTools } from '../utils';
 
+export type ToolsConfiguration = {
```

Comment:
why did you create 2 different types for this class and the MCP one? they're identical